theano/tensor/nnet/conv3d2d.py

```diff
@@ -117,6 +113,8 @@ class IncDiagonalSubtensor(Op):
     """
     The gradient of DiagonalSubtensor
     """
+    __props__ = ("inplace")
```

Comment:
Here as well, it has to be a tuple: `("inplace",)`. Same thing at other places in that PR.